crates/utils/src/rectangle.rs

```diff
@@ -149,6 +149,24 @@ impl Rectangle {
             self.set_height(point.y() - self.y());
         }
     }
+
+    /// Box itself inside another rectangle
+    pub fn box_into(&mut self, _box: Rectangle) {
```

Comment:
Can you remove the `_` please of `_box` so it indicates that the parameter is not used.

Reply:
But `box` is a keyword

Reply:
Yes your right. It is ok to use an other name?